src/libraries/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs

```diff
@@ -364,7 +364,12 @@ public static string ToSerialString(this byte[] serialBytes)
             return ToUpperHexString(serialBytes);
         }
 
-#if NETCOREAPP || NETSTANDARD2_1
+#if NET5_0_OR_GREATER
+        private static string ToUpperHexString(ReadOnlySpan<byte> ba)
+        {
+            return Convert.ToHexString(ba);
+        }
+#elif NETCOREAPP || NETSTANDARD2_1
```

Comment:
I think anything except in [System/Diagnostics/Activity.cs](https://github.com/dotnet/runtime/pull/92483/files#diff-1f21ee16d832bed839eb66474c6e2d0ee2d2775070643086b12f2c0aed014e5b) was failing on other platforms, did they?

Reply:
same here, `System.Security.Cryptography.Pkcs 8.0.0` is still targeting net46*.

Reply:
Well, these files doesn't included in .net framework builds, but it could built for netstandard https://github.com/dotnet/runtime/blob/40d9bd2ae3f06cdaff269a53f0c90d48dff7ed70/src/libraries/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj#L55-L58